docs/concepts/cluster-administration/addons.md

```diff
@@ -18,6 +18,7 @@ Add-ons in each section are sorted alphabetically - the ordering does not imply
 * [Contiv](http://contiv.github.io) provides configurable networking (native L3 using BGP, overlay using vxlan, classic L2, and Cisco-SDN/ACI) for various use cases and a rich policy framework. Contiv project is fully [open sourced](http://github.com/contiv). The [installer](http://github.com/contiv/install) provides both kubeadm and non-kubeadm based installation options.
 * [Flannel](https://github.com/coreos/flannel/blob/master/Documentation/kube-flannel.yml) is an overlay network provider that can be used with Kubernetes.
 *  [Nuage](https://github.com/nuagenetworks/nuage-kubernetes/blob/v5.1.1-1/docs/kubernetes-1-installation.rst) is an SDN platform that provides policy-based networking between Kubernetes Pods and non-Kubernetes environments with visibility and security monitoring.
+*  [Multus](https://github.com/Intel-Corp/multus-cni) is a Multi plugin for multiple network support in Kubernetes to support all CNI plugins (eg. contiv, cilium, Flannel, Calico), in addition to SRIOV, DPDK, OVS-DPDK and VPP based workloads in Kubernetes.
```

Comment:
@rkamudhan please move Multus in front of Nuage since we list plugins alphabetically :)
other nits: contiv --> Contiv, cilium --> Cilium.